kubernetes/cleanup

```diff
@@ -8,5 +8,6 @@ log "Cleaning up old containers"
 for i in `docker ps -a | grep k8s | awk '{print $1}'`; do docker rm -vf $i; done
 
 log "Cleaning the tmp and manifests folders"
-sudo rm -rf tmp/* manifests/* 
+sudo rm -rf tmp/* manifests/*
+sudo rm -rf /var/log/pods/*
```

Comment:
https://www.freedesktop.org/software/systemd/man/systemd-run.html
nvm about `systemd-exec` that's the execution env.